src/utils/axiosWithAuth.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const axiosWithAuth = () => {
+  const token = localStorage.getItem("token");
+
+  return axios.create({
+    baseURL: "https://api.groa.us/api/users",
+    headers: {
+      "Content-Type": "application/json",
```

Comment:
Do we need content-Type: application/json ?

Reply:
I don't think it's needed, it was here when I was cleaning up and didn't seem to do harm.